README.md

```diff
@@ -15,7 +15,7 @@ Requirements
 - python (2.7, 3.5, 3.6)
 - django (1.11, 2.0, 2.1)
 - djangorestframework (>=3.1, <4.0)
-- social-auth-core (>=1.7.0, <2.0)
+- social-auth-core (>=1.7.0, <3.0)
```

Comment:
Shouldn't this also be updated to <= 3.0?

Reply:
Good catch.. Just updated!